README.md

```diff
@@ -14,7 +21,162 @@ We call this the fuel mix, and this fuel mix can impact on the carbon intensity
+
+## grid-intensity CLI
+
+The CLI allows you to interact with carbon intensity data from multiple providers.
+
+```sh
+$ grid-intensity
+Provider ember-climate.org needs an ISO country code as a region parameter.
+ESP detected from your locale.
+ESP
+{
+	"country_code_iso_2": "ES",
+	"country_code_iso_3": "ESP",
+	"country_or_region": "Spain",
+	"year": 2021,
+	"latest_year": 2021,
+	"emissions_intensity_gco2_per_kwh": 193.737
+}
+```
+
+The `--provider` and `--region` flags allow you to select other providers and regions.
+You can also set the `GRID_INTENSITY_PROVIDER` and `GRID_INTENSITY_REGION` environment
+variables or edit the config file at `~/.config/grid-intensity/config.yaml`.
+
+```sh
+$ grid-intensity --provider carbonintensity.org.uk --region UK
+{
+	"from": "2022-07-14T14:30Z",
+	"to": "2022-07-14T15:00Z",
+	"intensity": {
+		"forecast": 184,
+		"actual": 194,
+		"index": "moderate"
+	}
+}
+```
+
+See providers section for how to configure other providers.
+
+## grid-intensity exporter
+
+The `exporter` subcommand starts the prometheus exporter on port 8000.
+
+```sh
+$ grid-intensity exporter --provider ember-climate.org --region FR
+Using provider "ember-climate.org" with region "FR"
+Metrics available at :8000/metrics
+```
+
+View the metrics with curl.
+
+```
+$ curl -s http://localhost:8000/metrics | grep grid
+# HELP grid_intensity_carbon_average Average carbon intensity for the electricity grid in this region.
+# TYPE grid_intensity_carbon_average gauge
+grid_intensity_carbon_average{provider="ember-climate.org",region="FR",units="gCO2 per kWh"} 67.781
+```
+
+### Docker Image
+
+Build the docker image to deploy the exporter.
+
+```sh
+CGO_ENABLED=0 GOOS=linux go build -o grid-intensity .
+docker build -t thegreenwebfoundation/grid-intensity:latest .
+```
+
+### Kubernetes
+
+Install the [helm](https://helm.sh/) chart in [/helm/grid-intensity-exporter](https://github.com/thegreenwebfoundation/grid-intensity-go/tree/main/helm/grid-intensity-exporter).
+Needs the Docker image to be available in the cluster.
+
+```sh
+helm install --set gridIntensity.region=FR grid-intensity-exporter helm/grid-intensity-exporter
+```
+
+### Nomad
+
+Edit the Nomad job in [/nomad/grid-intensity-exporter.nomad](https://github.com/thegreenwebfoundation/grid-intensity-go/blob/main/nomad/grid-intensity-exporter.nomad) to set the
+env vars `GRID_INTENSITY_REGION` and `GRID_INTENSITY_PROVIDER`
+
+Start the Nomad job. Needs the Docker image to be available in the cluster.
+
+```sh
+nomad run ./nomad/grid-intensity-exporter.nomad
+```
+
+## grid-intensity-go library
+
+See the [/examples/](https://github.com/thegreenwebfoundation/grid-intensity-go/tree/main/examples) 
+directory for examples of how to integrate each provider.
+
+## Providers
+
+Currently these providers of carbon intensity data are integrated. If you would like
+us to integrate more providers please open an [issue](https://github.com/thegreenwebfoundation/grid-intensity-go/issues).
```

Comment:
@mrchrisadams For adding providers is opening an issue the best route? Or is the support form better?

I'm fine to change this or drop it if it isn't needed.

Reply:
hmm… issues are good, but not everyone will want to ask things publicly, and in the case of people wanting help implementing carbon aware scheduling on a system they have, it would be good to provide a channel for them to ask for help and pointers.

Maybe something like this:

> If you have any other inquiries, or you're interested in having this extended to fit your use case on a commercial basis please [contact us via our support form](https://www.thegreenwebfoundation.org/support-form/)